tests/run-make/const-prop-lint/Makefile

```diff
@@ -1,9 +0,0 @@
-include ../tools.mk
-
-# Test that emitting an error because of arithmetic
-# overflow lint does not leave .o files around
-# because of interrupted codegen.
-
-all:
-	$(RUSTC) input.rs; test $$? -eq 1
-	ls *.o; test $$? -ne 0
```

Comment:
Just a comment: I would need to do some heavy mental gymnastics to understand what `ls *.o; test $$? -ne 0` was doing

Reply:
We ensure there was no files with the `.o` extension (yep, took me a while to understand that too ^^').